packages/jsii-rosetta/test/commands/transliterate.test.ts

```diff
@@ -0,0 +1,917 @@
+                Object {
+                  "name": "props",
+                  "type": Object {
+                    "fqn": "testpkg.ClassNameProps",
+                  },
+                },
+              ],
+            },
+            "interfaces": Array [
+              "testpkg.IInterface",
+            ],
+            "kind": "class",
+            "locationInModule": Object {
+              "filename": "index.ts",
+              "line": 40,
+            },
+            "methods": Array [
+              Object {
+                "docs": Object {
+                  "example": "// Example automatically generated. See https://github.com/aws/jsii/issues/826
+      ClassName.StaticMethod();",
+                  "remarks": "It can be invoked easily.",
+                  "summary": "A static method.",
+                },
+                "locationInModule": Object {
+                  "filename": "index.ts",
+                  "line": 46,
+                },
+                "name": "staticMethod",
+                "parameters": Array [
+                  Object {
+                    "name": "_enm",
+                    "optional": true,
+                    "type": Object {
+                      "fqn": "testpkg.EnumType",
+                    },
+                  },
+                ],
+                "static": true,
+              },
+              Object {
+                "docs": Object {
+                  "summary": "An instance method call.",
+                },
+                "locationInModule": Object {
+                  "filename": "index.ts",
+                  "line": 61,
+                },
+                "name": "methodCall",
+                "overrides": "testpkg.IInterface",
+              },
+            ],
+            "name": "ClassName",
+            "properties": Array [
+              Object {
+                "docs": Object {
+                  "summary": "A property value.",
+                },
+                "locationInModule": Object {
+                  "filename": "index.ts",
+                  "line": 50,
+                },
+                "name": "property",
+                "overrides": "testpkg.IInterface",
+                "type": Object {
+                  "fqn": "testpkg.EnumType",
+                },
+              },
+            ],
+          },
+          "testpkg.ClassNameProps": Object {
+            "assembly": "testpkg",
+            "datatype": true,
+            "fqn": "testpkg.ClassNameProps",
+            "kind": "interface",
+            "locationInModule": Object {
+              "filename": "index.ts",
+              "line": 35,
+            },
+            "name": "ClassNameProps",
+            "properties": Array [
+              Object {
+                "abstract": true,
+                "immutable": true,
+                "locationInModule": Object {
+                  "filename": "index.ts",
+                  "line": 37,
+                },
+                "name": "foo",
```

Comment:
Are these inline snapshots actually update? I never managed to get that to work..

Also not sure that's better. It really pollutes the source

Reply:
They do update, yes (I have iterated over those a bit).

Best practice is to treat the snapshots as code... I agree that they take up quite a lot of space, though.... But I find they are more difficult to inspect when they're in a different file... I don't feel super strongly about this though... If you'd rather I use "regular" snapshots, it's easy to flip to that.